.github/workflows/zig.yml

```
@@ -0,0 +1,25 @@
+name: CI - Zig
+
+on:
+  create:
+  push:
+    branches: master
+  pull_request:
+  schedule:
+    - cron: "0 18 * * *"
+  workflow_dispatch:
+
+concurrency:
+  group: ci-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: mlugg/setup-zig@v1
```

```suggestion
      - uses: mlugg/setup-zig@v2
```

I believe the new version just got released.  https://github.com/mlugg/setup-zig/commit/aa9ad5c14eb3452e235a441c4f9a8e89f20d97bd